package.json

```diff
@@ -44,12 +44,17 @@
     "broadcastchannel-polyfill": "^1.0.0",
     "history": "^4.9.0",
     "marksy": "^8.0.0",
+    "merge-anything": "^3.0.3",
```

Comment:
Was taking a peek at this and they just updated to `3.0.4` so might just want to catch it, but no biggie.